app/src/main/res/layout/previous_responses_header_item.xml

```diff
@@ -59,7 +59,7 @@
       android:paddingEnd="8dp"
       android:text="@{viewModel.computePreviousResponsesHeaderText()}"
       android:textAllCaps="true"
-      android:textColor="@color/color_def_mid_grey"
+      android:textColor="@color/component_color_question_player_activity_text_color"
```

Comment:
Ditto as above. Same applies for all other colors in this file.

Reply:
Done